interfaces/cython/cantera/soln2ck.py

```diff
@@ -0,0 +1,362 @@
+"""writes a solution object to a chemkin inp file
+
+currently only works for Elementary, Falloff and ThreeBody Reactions
+Cantera development version 2.3.0a2 required
+"""
+from __future__ import print_function
+from __future__ import division
+
+import os
+from string import Template
+
+import cantera as ct
+from cantera.test.test_mechanism_from_solution import test
+import ck2cti
+
+
+def write(solution):
+    """Function to write cantera solution object to inp file.
+
+    :param solution
+        Cantera solution object
+
+    :return
+        Name of trimmed Mechanism file (.inp)
+
+    >>> soln2ck.write(gas)
+    """
+    trimmed_solution = solution
+    input_file_name_stripped = trimmed_solution.name
+    cwd = os.getcwd()
+    output_file_name = os.path.join(cwd,
+                                    'pym_' +
+                                    input_file_name_stripped +
+                                    '.inp')
+    f = open(output_file_name, 'w+')
+
+    #Work functions
+
+    calories_constant = 4184.0 #number of calories in 1000 Joules of energy
+    def eliminate(input_string, char_to_replace, spaces='single'):
```

Comment:
It does perform the same function as `replace`, however I had a few strings which needed 4 or 5 different characters removed or replaced. 

I didn't think `str.replace().replace..` etc would be appropriate, so I moved it to the `eliminate` function. I suppose I could wrap the list of characters to replace in a for loop and then do `str.replace()` if that would be more appropriate?